conf.json.template

```diff
@@ -13,5 +13,13 @@
 	"http.port":8080,
 	"http.host":"localhost",
 	"http.endpoint":"/api/v3",
-	"http.port.management":13371
+	"http.port.management":13371,
+	"salt": "cyface_salt",
+	"upload.expiration": 600,
```

Comment:
You probably meant 10 minutes, i.e. `1000 * 60 * 10` ?

Why did you change the default upload expiration time of 1 week?

```
/**
     * The default number of milliseconds to wait before removing cached uploads after last modification: 7 days.
     */
    private static final long DEFAULT_UPLOAD_EXPIRATION_TIME = 1000 * 60 * 60 * 24 * 7;
```